### DISCUSSION TITLE
How can I obtain all the features of TailwindCSS in UnoCSS?

### DISCUSSION THREAD

**Discussion by cesswhite** [unocss/unocss#2414] [Q&A]
I mean, is UnoCSS able to match the features of TailwindCSS automatically or with every update of TailwindCSS we would have to wait for a PR to get its functionalities?

Today they released version 3.3 which has several improvements in terms of the gradient system and more colors. That's why my question

**Comment by jbreuil** [comment]
UnoCSS is an engine, it is already able to replicate any features that Tailwind could add.
What you are asking is related to the preset-uno which is plugged on preset-windy. This preset is adding rules in order to replicate windiCSS, so to get the latest features of tailwind as predefined rules you would need to wait that someone made a PR on this preset.
You can add custom rules in your config if you need them quickly.
